invenio_app_ils/vocabularies/data/author_affiliation_identifier_schemes.json

```diff
@@ -0,0 +1,107 @@
+[
+  {
+    "type": "author_affiliation_identifier_scheme",
+    "key": "ContactPerson",
+    "text": "Contact Person"
+  },
+  {
+    "type": "author_affiliation_identifier_scheme",
+    "key": "DataCollector",
+    "text": "Data Collector"
+  },
+  {
+    "type": "author_affiliation_identifier_scheme",
+    "key": "DataCurator",
+    "text": "Data Curator"
+  },
+  {
+    "type": "author_affiliation_identifier_scheme",
+    "key": "DataManager",
+    "text": "Data Manager"
+  },
+  {
+    "type": "author_affiliation_identifier_scheme",
+    "key": "Distributor",
+    "text": "Distributor"
+  },
+  {
+    "type": "author_affiliation_identifier_scheme",
+    "key": "Editor",
+    "text": "Editor"
+  },
+  {
+    "type": "author_affiliation_identifier_scheme",
+    "key": "HostingInstitution",
+    "text": "Hosting Institution"
```

Comment:
isn't this one the `author type`? https://schema.datacite.org/meta/kernel-4.3/include/datacite-contributorType-v4.xsd